### DISCUSSION TITLE
Welcome to Hello-World- Discussions!

### DISCUSSION THREAD

**Discussion by Goldenboy3023** [Goldenboy3023/Hello-World-#4] [Q&A] [answered]
How do I go about adding the forEach? Can you give an example.

**Comment by onthegocode** [comment, marked as answer]
So forEach is a Method for Arrays, so what you can do as a test is create an array that has 10 elements in it and then uses the forEach to loop over it. Just as a test then I will show you how to use the DOM.

**Comment by onthegocode** [comment]
Here's an example: 
`const arr = [1,2,3,4,5,6,7,8,9,10];
arr.forEach(el => console.log('Hello World');
`
